format-delimited/src/test/java/io/cdap/plugin/format/delimited/common/DataTypeDetectorUtilsTest.java

```diff
@@ -38,47 +38,63 @@ public class DataTypeDetectorUtilsTest {
   @Test
   public void testExtractedColumnNames() {
     String headerLine = "column_A;column_B;column_C";
-    String[] actualColumnNames = DataTypeDetectorUtils.setColumnNames(headerLine, true, false, ";");
+    String[] actualColumnNames = DataTypeDetectorUtils.setColumnNames(headerLine, true, false, ";", null);
```

Comment:
tests should be added to the format test classes (AvroInputFormatProviderTest, DelimitedInputFormatProviderTest, etc)